games-fps/fteqw/fteqw-20240406~git.recipe

```diff
@@ -0,0 +1,102 @@
+- Vast amount of map, model & image formats are supported.
+- Advanced console, with descriptions & autocompletion.
+- Plugin support, enabling use of FFMPEG, Bullet/ODE physics & more.
+- Extensive suite of QuakeC/entity debugging features.
+- Deep integration with FTEQCC, which can even be executed in-game.
+- Support for split-screen local multiplayer.
+- Voice-chat via Opus & Speex.
+- Support for hundreds of players on a single server.
+- Works on Windows, Linux, OpenBSD, Haiku... & more!
+- New features are added all the time in cooperation with modders!"
+HOMEPAGE="https://www.fteqw.org/"
+COPYRIGHT="2002-2023 Forethought Entertainment"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitRevision="1f9f3635f0aef3b2eed6b40e35fcf6223c6ad533"
+SOURCE_URI="https://github.com/fte-team/fteqw/archive/$gitRevision.tar.gz"
+CHECKSUM_SHA256="c9376b7516444cc12e04a771f6e3bad91146762c3ecadc2aeda87287922dc48e"
+SOURCE_DIR="fteqw-$gitRevision"
+ADDITIONAL_FILES="fteqw-$portVersion.rdef.in"
+PATCHES="fteqw-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fteqw$secondaryArchSuffix = $portVersion
+	cmd:fteqw
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:strip
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DFTE_USE_SDL=true \
+		-DFTE_LIBS=network
+	make -Cbuild fteqw $jobArgs
+	strip build/fteqw
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp -a build/fteqw $binDir/fteqw
+
+	local APP_SIGNATURE="application/x-vnd.FTEQW"
+	local MAJOR="`echo "${portVersion//~git}" | cut -d. -f1`"
+	local MIDDLE="`echo "${portVersion//~git}" | cut -d. -f2`"
+	local MINOR="`echo "${portVersion//~git}" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/fteqw-$portVersion.rdef.in > fteqw-$portVersion.rdef
+
+	addResourcesToBinaries fteqw-$portVersion.rdef $binDir/fteqw
+	mimeset -f $binDir/fteqw
+
+	addAppDeskbarSymlink $binDir/fteqw "FTEQW"
```

Comment:
I would drop the Deskbar link, this is intened to be launched from Tracker, launching it from the Deskbar is pretty pointless at this stage. :)